src/spikeinterface/core/template.py

```diff
@@ -0,0 +1,196 @@
+import numpy as np
+import json
+from dataclasses import dataclass, field, astuple
+from .sparsity import ChannelSparsity
+
+
+@dataclass
+class Templates:
+    """
+    A class to represent spike templates, which can be either dense or sparse.
+
+    It is constructed with the following parameters:
+    ----------
```

Comment:
```suggestion
    Parameters
    ----------
```
I think you would still just say parameters here right? Just for cleanliness/lack of verbosity. But I'm not super against having the statement there.

Reply:
Yes, this is fine for me, thanks.